command/commands/team.py

```diff
@@ -211,5 +212,5 @@ def create_helper(self,
             return msg, 200
         except GithubAPIException as e:
             logging.error("team creation unsuccessful")
-            return "Team creation unsuccessful with the following error"\
+            return "Team creation unsuccessful with the following error" \
```

Comment:
```suggestion
return f"Team creation unsuccessful with the following error: {e.data}", 200
```

Reply:
I would make this change but I believe a number of these functions have similar issues. Do you mind making a new ticket to fix them, so we can do them all in one sweep?